www/install/createTables.sql

```diff
@@ -1630,6 +1630,7 @@ CREATE TABLE `nagios_server` (
   `engine_version` varchar(255) DEFAULT NULL,
   `centreonbroker_logs_path` VARCHAR(255),
   `remote_id` int(11) NULL,
+  `remote_server_centcore_ssh_proxy` enum('0','1') NULL DEFAULT '1',
```

Comment:
as the default value is set to '1' and we're introducing strict mode, shouldn't it be more adequate to choose the NOT NULL arg instead of NULL ?